components/eam/cime_config/config_component.xml

```diff
@@ -107,21 +107,21 @@
       <value compset="RCP4_EAM"                         >2006-2100_cam5_rcp45</value>
       <value compset="RCP6_EAM"                         >2006-2100_cam5_rcp60</value>
       <value compset="RCP8_EAM"                         >2006-2100_cam5_rcp85</value>
-      <value compset="1850(?:SOI)?_EAM.*CMIP6"          >1850_eam_CMIP6</value>
-      <value compset="1850(?:SOI)?_EAM.*CMIP6-1pctCO2"  >1850_eam_CMIP6-1pctCO2</value>
+      <value compset="1850(?:SOI)?_EAM.*CMIP6"          >1850_eam_CMIP6_chemUCI-Linoz-mam5-vbs</value>
+      <value compset="1850(?:SOI)?_EAM.*CMIP6-1pctCO2"  >1850_eam_CMIP6_chemUCI-Linoz-mam5-vbs-1pctCO2</value>
```

Comment:
why not just edit the existing 1850_eam_CMIP6.xml file instead of creating new use cases?   Does the "1850_eam_CMIP6" case still have any value?

Reply:
Agreed, @rljacob . After his PR, there would be no compset directly using the original use_case file. However, for now, we have to use a different use_case filename like what shown, because build-namelist literally parses for the substring of 'chemUCI-Linoz' to specify the related forcing settings. This is introduced by a v3atm PR and is not an ideal way. Look to implement a new flag as a replacement.